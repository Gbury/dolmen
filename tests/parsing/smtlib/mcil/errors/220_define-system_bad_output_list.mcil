(define-system sys :output (Int))