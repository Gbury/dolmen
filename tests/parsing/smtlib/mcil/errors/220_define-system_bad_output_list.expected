File "tests/parsing/smtlib/mcil/errors/220_define-system_bad_output_list.mcil", line 1, character 28-31:
1 | (define-system sys :output (Int))
                                ^^^
Error while parsing a system output variable declaration,
      read the symbol 'Int',
      but expected a sorted variable of the form "(var sort)", or a closing
      parenthesis.
